sections/collapsible-content.liquid

```diff
@@ -1,8 +1,22 @@
 {{ 'component-accordion.css' | asset_url | stylesheet_tag }}
 {{ 'collapsible-content.css' | asset_url | stylesheet_tag }}
 
+{%- style -%}
+  .section-{{ section.id }}-padding {
+    padding-top: calc({{ section.settings.padding_top }}px * 0.75);
+    padding-bottom: calc({{ section.settings.padding_bottom }}px  * 0.75);
+  }
+
+  @media screen and (min-width: 750px) {
+    .section-{{ section.id }}-padding {
+      padding-top: {{ section.settings.padding_top }}px;
+      padding-bottom: {{ section.settings.padding_bottom }}px;
+    }
+  }
+{%- endstyle -%}
+
 <div class="collapsible-content collapsible-{{ section.settings.layout }}-layout color-{{ section.settings.color_scheme }}{% if section.settings.layout == 'section' %} page-width{% endif %}">
-  <div class="collapsible-content__wrapper{% if section.settings.layout == 'section' %} color-{{ section.settings.container_color_scheme }}{% endif %}">
+  <div class="collapsible-content__wrapper{% if section.settings.layout == 'section' %} color-{{ section.settings.container_color_scheme }}{% endif %} section-{{ section.id }}-padding">
```

Comment:
Small nitpick, it's nicer to have the classes that are always on the element before the if statement I think. We tend to do it this way across the theme.